src/development/tools/sdk/release-notes/release-notes-1.12.13.md

```diff
@@ -2379,7 +2379,7 @@ We added the ability to get id in the callback in the Android Alarm Manager plug
 
 #### Plugin: Android Intent
 
-We made several improvements in the Android Intent plugin including adding the ability to pass intent flags (contributed by a community member!), and upgrading it to the [new plugin API](https://github.com/flutter/website/blob/staging-add-to-app-do-not-delete-until-2020/src/docs/development/packages-and-plugins/plugin-api-migration.md).
+We made several improvements in the Android Intent plugin including adding the ability to pass intent flags (contributed by a community member!), and upgrading it to the [new plugin API](https://github.com/flutter/website/blob/staging-add-to-app-do-not-delete-until-2020/sr/development/packages-and-plugins/plugin-api-migration.md).
```

Comment:
Find and replace mistake? Looks like it took `src/docs` replaced `/docs` and ended up with `sr`

Reply:
Yes! Just pushed those changes up.

Reply:
Thanks Brian! Looks like `https://github.com/flutter/website/blob/staging-add-to-app-do-not-delete-until-2020/src/development/packages-and-plugins/plugin-api-migration.md` doesn't exist anymore, John pointed me over to `https://github.com/flutter/website/blob/main/src/development/packages-and-plugins/plugin-api-migration.md` which is the correct link (assuming the new director structure without `/docs`). Can we replace for all 3 occurrences? Sorry missed that on the first review. 

Actual live link right now: `https://github.com/flutter/website/blob/main/src/docs/development/packages-and-plugins/plugin-api-migration.md`